collector/loadavg_linux.go

```diff
@@ -16,37 +16,20 @@
 package collector
 
 import (
-	"fmt"
-	"io/ioutil"
-	"strconv"
-	"strings"
+	"syscall"
 )
 
-// Read loadavg from /proc.
 func getLoad() (loads []float64, err error) {
-	data, err := ioutil.ReadFile(procFilePath("loadavg"))
-	if err != nil {
-		return nil, err
-	}
-	loads, err = parseLoad(string(data))
-	if err != nil {
+	const scale float64 = 65536 // LINUX_SYSINFO_LOADS_SCALE
```

Comment:
is this the same on all architectures and kernel versions?

Reply:
We solved this elsewhere[0] by using a `C` reference. But this const seems to come from BSD sources.  In looking at the source for `uptime` in `procps`, it reads from `/proc/`[1].

[0]: https://github.com/prometheus/node_exporter/pull/846
[1]: https://gitlab.com/procps-ng/procps/blob/master/proc/sysinfo.c#L444-461